packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -33,7 +33,8 @@ export default function LetterSpacingControl( {
 	} );
 	return (
 		<UnitControl
-			label={ __( 'Letter-spacing' ) }
+			className="block-editor-letter-spacing-control"
```

Comment:
I don't think we need this anymore after https://github.com/WordPress/gutenberg/pull/36385/commits/5f7626d21641f3cb6bff6ea2e8c892ea5df36dac right?

Reply:
Thanks for spotting.